### ISSUE TITLE
[Haxe] Haxe Feathers UI Project fail to detect browser close after Build & Run

### ISSUE THREAD

**Issue by rat-moonshine** [prominic/Moonshine-IDE#666]
It appears that only HTML5 platform target is functioning, at this moment (?) I reported with error for Windows target while a test.

While running a project with HTML5 target, it successfully completes the compilation and runs on Chrome browser. I'm not sure if this run is limited to Chrome because Mozilla was my default browser. 

1. Perform "Build & Run" to Haxe Feathers UI project
2. Moonshine starts Chrome debugger running the application eventually
3. Following marker appeared at bottom-status in Moonshine:
![image](https://user-images.githubusercontent.com/24370870/75153838-59839b80-5732-11ea-82df-019d84503749.png)
4. Close the Chrome browser manually, the above marker remains to Moonshine status bar 
5. Clicking on the STOP icon in said marker throws the following error:
```
: Launching Haxe project...
: Command: C:\MoonshineSDKs\NodeJS\node-v12.15.0\node.exe C:\Program Files (x86)\Moonshine\elements\http-server\bin\http-server C:\Users\Devsena\Desktop\NewHaxeFeathersUIProject\bin\html5\bin -p 3000
: Starting up http-server, serving C:\Users\Devsena\Desktop\NewHaxeFeathersUIProject\bin\html5\bin
: Available on:
:   http://192.168.0.100:3000
:   http://127.0.0.1:3000
: Hit CTRL-C to stop the server
: HTTP server has been terminated.
: Process error: ERROR: The process "17448" not found.
```

**Comment by piotrzarzycki21** [comment]
@rat-moonshine I believe this is [the reason](https://github.com/prominic/Moonshine-IDE/issues/664#issuecomment-589741895) of this issue and we rather doesn't have too much influence on that - other than wait.

**Comment by joshtynjala** [comment]
microsoft/vscode-chrome-debug#963

**Comment by rat-moonshine** [comment]
> @rat-moonshine I believe this is [the reason](https://github.com/prominic/Moonshine-IDE/issues/664#issuecomment-589741895) of this issue and we rather doesn't have too much influence on that - other than wait.



> microsoft/vscode-chrome-debug#963

Ah! Good to hear that the issue already raised in past. Sorry, I see I'm a bit outdated on this regard )

I can close this issue for now as the problem already discussed, and we agreed to wait on newer extension release.

Is the Chrome Debug Adapter also includes a run-able instance of Chrome browser? What would happen if user don't have Chrome installed? Also, why it opens Chrome in case of 'Build & Run' and when I have different default browser (Mozilla)?

**Comment by joshtynjala** [comment]
No, Chrome browser is not bundled. I don't know exactly what happens when Chrome is not installed on the user's system. Most likely, the debug adapter will fail to find Chrome, and an error message will be returned to Moonshine. Or perhaps the attempt to connect will simply time out. Either way, there's probably some indication to the user that something has gone wrong. However, I have not tested it personally.

vscode-chrome-debug supports Chrome only, so it simply cannot launch a different browser. With that in mind, your default browser doesn't matter. There's a separate debug adapter for Firefox, which we could also potentially integrate into Moonshine. Then, we could allow the user to choose which browser that they prefer to use for debugging in Moonshine. Probably in the run settings, similar to how they choose an AIR simulator.

**Comment by rat-moonshine** [comment]
Okay. That sounds would be more flexible for the users having Mozilla as default browser. If Mozilla adapter is similar to Chrome, we may want to restrict by supporting only Chrome debugger, too; An user may have other different browser set as default.
